packages/aws-cdk-lib/aws-lambda/lib/runtime.ts

```diff
@@ -113,7 +113,7 @@ export class Runtime {
    * The latest NodeJS version currently available in ALL regions (not necessarily the latest NodeJS version
    * available in YOUR region).
    */
-  public static readonly NODEJS_LATEST = new Runtime('nodejs18.x', RuntimeFamily.NODEJS, { supportsInlineCode: true, isVariable: true });
+  public static readonly NODEJS_LATEST = new Runtime('nodejs20.x', RuntimeFamily.NODEJS, { supportsInlineCode: true, isVariable: true });
```

Comment:
Instead of hardcoding node20 in the many places we have 18 currently hardcoded, can we just use this?

Reply:
Along with introducing NODEJS_LATEST in resource framework, modified this in alpha modules and other places in this PR to point to this except in Readme above(assuming documentation should reflect the latest version being supported).